libs/ledger-live-common/.unimportedrc.json

```diff
@@ -367,6 +372,7 @@
   "ignoreUnused": [
     "@dfinity/agent",
     "@ledgerhq/hw-app-eth",
+    "@ledgerhq/hw-app-solana",
```

Comment:
we probably want to remove the dependency in common if we moved it to the package